src/access/AccessControl.h

```diff
@@ -18,6 +18,12 @@
 
 #pragma once
 
+#include <access/AccessConfig.h>
```

Comment:
should this be wrapped in the #define too?

Reply:
No, since that's where the #define is #defined :-)